src/Deserialization/Transaction.js

```diff
@@ -1,3 +1,10 @@
+/* global exports */
```

Comment:
This is a more general comment, but I recall when the `Deserialization.Transaction` module was first introduced we wanted to reduce the amount of logic written in JS